sources/packages/web/src/services/AuthService.ts

```diff
@@ -93,6 +110,20 @@ export class AuthService {
       });
 
       if (this.keycloak.authenticated) {
+        // In case of user closed browser without logout.
+        window.addEventListener("beforeunload", this.logUserClosedBrowser);
+        if (sessionStorage.getItem(USER_CLOSED_BROWSER) === "true") {
+          // In case of user reopened browser with an active session.
+          await ApiClient.AuditApi.audit(AuditEvent.BrowserReopened);
```

Comment:
I see that this only gets called when the browser is reopened for the first time after closing. 2nd time onwards, it is not getting called.

Reply:
I think it is because on closing the tab, the sessionStorage is cleared vs it is not cleared on tab/window reload.

Reply:
You are right. In my tests I used CTRL+SHIFT+T that reopens the browser but that also keeps the sessionStorage vars. When we close the tab and open another tab the sessionStorage is fresh and it won't find the var. I'm using localStorage for this one. Thanks!